app/views/checklist_tests/_field_values.html.erb

```diff
@@ -3,21 +3,102 @@
 # local variables
 #
 #   field_values   (Hash)
+#   product_test (ChecklistTest)
```

Comment:
Also since you added the local variable `product_test`, you need to pass in a value for that whenever this partial is rendered. Right now the site breaks if you click the "Download Criteria List" button on the checklist test page, because `product_test` is undefined. You should set the appropriate value for `product_test`[there](https://github.com/projectcypress/cypress/blob/code_lookup/app/views/checklist_tests/print_criteria.html.erb#L48), and then download the criteria list and make sure that looks how it should.